### PR TITLE
fix: add defaults for optional extractor configs

### PR DESCRIPTION
**Pull request by louise-poole**
Tested - works like a charm ✨

### CODE REVIEW

File: extractors.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ extractors:
     protocol_types:
       - name: "sushiswap_v2_pool"
         financial_type: "Swap"
-    spkg: "substreams/ethereum-uniswap-v2/ethereum-sushiswap-v2-v0.2.0.spkg"
+    spkg: "substreams/ethereum-sushiswap-v2/substreams-ethereum-sushiswap-v2-v0.2.0.spkg"
```
Comment (zizou0x) [inline extractors.yaml]: We usually name them like this in S3, I'd like to keep the naming consistent

```suggestion
    spkg: "substreams/ethereum-sushiswap-v2/ethereum-sushiswap-v2-v0.2.0.spkg"
```

Comment (kayibal) [inline extractors.yaml]: Why do we need a different package here?

File: tycho-indexer/src/extractor/runner.rs [hunk 2]
```diff
@@ -281,10 +281,16 @@ pub struct ExtractorConfig {
     protocol_types: Vec<ProtocolTypeConfig>,
     spkg: String,
     module_name: String,
+    #[serde(default)]
     pub initialized_accounts: Vec<Bytes>,
+    #[serde(default = "default_initialized_accounts_block")]
```
Comment (zizou0x) [inline tycho-indexer/src/extractor/runner.rs]: Can't we just `serde(default)` this one too? I'd expect `i64` to default to 0

File: extractors.yaml (hunk 1 above)
Comment (louise-poole) [reply extractors.yaml]: Because the package has sushiswap specific info (factory address etc)

**Pull request merged**